### PR TITLE
Milestone: Deployment

### PR DESCRIPTION
**Pull request by didymus707**
Project is deployed on heroku
Readme has been updated to hold that information

### CODE REVIEW

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work @didymus707 :clap: Almost done. However, some changes need to be made in order to get approval for this milestone. Check :arrow_down: 

- **[REQUIRED]** The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
  - Several sections are empty (TBA).
  - You could add screenshots of the app

### That' it :relieved: When finished, kindly submit a code re-review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

File: README.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
 # Scaffold for social media app with Ruby on Rails
 
-> This repo includes intial code for social media app with basic styling. Its purpose is to be a starting point for Microverse students.
+> This purpose of this project is to practice all skills and techniques learnt during the course of Rails to build this social media application. And in this project, features like, creating posts, liking a post, adding friends were implemented.
```
Comment (euqueme) [inline README.md]: - [ ] The should say `The purpose...` instead of `This purpose...` 

- [ ] change `learnt` for `learned`

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @didymus707 :lollipop: 

#### Thank you for doing the changes requested by the previous reviewer :medal_sports: you've done a great job with this project and you should feel proud of all your hard work :muscle:  please do the following changes :construction: :construction_worker: 

#### README

- [ ] I've left comments in your Readme file where I found some typos let's fix them :point_down: 

- [ ] You deleted the Run test section that explain how to run your test cases from your Readme let's bring it back.

- [ ] You also deleted the Deployment section let's bring it back and put information about the deployment (you can say you used Heroku to deploy your app and put the link to the live demo again)

## [OPTIONAL]

> The next are requirements from previous milestones that either weren't mentioned by previous reviewers or weren't fully accomplished so I sincerely apologize for requesting them in this milestone but I strongly recommend working on them.

#### Milestone 5

- [ ] There is a button to invite a user to friendship on users list page and on single-user page ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/user_can_see_invite_buttton.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/))

- You have this button only in the user's list page but it's missing in the user's show page, let's add it

#### General

##### No Logic in Views or partials

- [ ] There should be no if, if/else, unless or ternary operators straight in the views this complexity should be handled by helper methods, please read these articles that explain with examples how to remove complexity from views and partials [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)  and [Rails View Helpers — Best Practices Guide](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4), these are the views you need to fix:

- app/views/partials/_friends.html.erb
- app/views/partials/_pending_requests.html.erb
- app/views/partials/_rr_partial.html.erb

### That's it! :raised_hands: 

### Please :pray: fix all mandatory changes :technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Pull request merged**